interfaces/HO-Portal/src/app/users/users.page.html

```diff
@@ -1,6 +1,121 @@
+          class="users-title"
+        >
+          <strong> {{ 'page.users.pageTitle' | translate }} </strong>
+        </h2>
+        <p class="ion-no-margin">
+          {{ 'page.users.page-description' | translate }}
+        </p>
+      </ion-col>
+    </ion-item>
 
-<ion-content class="ion-padding">
-  <h2>{{ 'page.users.pageTitle'|translate }}</h2>
-  <p>{{ 'page.help.user-manual-content'|translate }}</p>
+    <!--tabs-->
+    <ion-segment
+      (ionChange)="segmentChanged($event)"
+      value="users"
+      class="tabs"
+    >
+      <ion-segment-button
+        value="users"
+        class="tab"
+      >
+        <ion-label
+          ><strong> {{ 'page.users.users-tab' | translate }}</strong></ion-label
+        >
+      </ion-segment-button>
+      <ion-segment-button
+        value="roles"
+        class="tab"
+      >
+        <ion-label
+          ><strong>{{ 'page.users.roles-tab' | translate }}</strong></ion-label
+        >
+      </ion-segment-button>
+    </ion-segment>
+
+    <!--tab 1 users-->
+    <div
+      *ngIf="segment === 'users'"
+      class="ion-padding"
+    >
+      <!--filter and add new user button-->
+      <div class="filter-and-button">
+        <!--filter-->
+        <ion-item
+          lines="none"
+          style="border: 1px solid #d1d4d7; border-radius: 4px; height: 40px"
+          class="ion-align-items-start filter"
+        >
+          <ion-icon
+            name="search"
+            [attr.title]="
+            'page.users.search'
+              | translate
+          "
+            aria-hidden="true"
+            size="small"
+            slot="start"
+            class="ion-margin-end"
+            style="height: 40px; margin-top: 0px; margin-bottom: 0opx"
+            color="black"
+          ></ion-icon>
+          <ion-input
+            [placeholder]="
+            'page.users.search'
+              | translate
+          "
+            [attr.title]="
+            'page.users.search'
+              | translate
+          "
+            [attr.aria-label]="
+            'page.users.search'
+              | translate
+          "
+            (ionInput)="onInput($event.detail.value)"
+            [debounce]="1000"
+            [clearInput]="true"
+            class="filter-input"
+            data-cy="table-text-filter"
+          >
+          </ion-input>
+        </ion-item>
+
+        <!--add new user button-->
+        <ion-button
+          shape="round"
+          fill="outline"
+          class="button"
+          [disabled]="true"
+          item-end
+          ><strong
+            >{{ 'page.users.add-new-user' | translate}}</strong
+          ></ion-button
+        >
+      </div>
+
+      <!--users table-->
+      <div class="ion-padding">
+        <app-users-table [filterValue]="filterInput"></app-users-table>
+      </div>
+    </div>
+
+    <!--tab 2 roles-->
+    <div
+      *ngIf="segment === 'roles'"
+      class="ion-padding"
+    >
+      <ion-text>Empty</ion-text>
```

Comment:
This text needs to be translatable.

Reply:
'roles' is a segment value not text 
'Empty' will be replaced with roles and permissions cards, it is ready, but need to merge this first, because of the tabs.

Reply:
Ah, yes. So its just a placeholder-during-development? 👍